### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by OnsJannet** [community/community#66807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So i tried opening my github account from a new computer i messed up with the password at first but i ended up changing the password and now i was able to log to my account however when i tried to clone a repo i got this error : You are marked as spam, and therefore cannot authorize a third party application. and i'm not sure what to do.

**Comment by cehfisher** [comment]
Hi @OnsJannet - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
